pkg/varlinkapi/config.go

```diff
@@ -1,14 +1,23 @@
 package varlinkapi
 
-import "github.com/projectatomic/libpod/cmd/podman/ioprojectatomicpodman"
+import (
+	"github.com/projectatomic/libpod/cmd/podman/ioprojectatomicpodman"
+	"github.com/urfave/cli"
+)
 
 // LibpodAPI is the basic varlink struct for libpod
 type LibpodAPI struct {
+	Cli *cli.Context
 	ioprojectatomicpodman.VarlinkInterface
 }
 
-var (
-	lp = LibpodAPI{}
-	// VarlinkLibpod instantiation
-	VarlinkLibpod = ioprojectatomicpodman.VarlinkNew(&lp)
-)
+//var (
+//	lp = LibpodAPI{}
+//	// VarlinkLibpod instantiation
```

Comment:
Can you remove the commented-out code? It looks like it's been replaced by New

Reply:
done